autoload/asyncomplete.vim

```diff
@@ -55,7 +58,7 @@ function! asyncomplete#register_source(info) abort
         execute 'augroup asyncomplete_source_event_' . a:info['name']
         for l:event in a:info['events']
             let l:exec =  'if get(b:,"asyncomplete_enable",0) | call s:python_cm_event("' . a:info['name'] . '", "'.l:event.'",asyncomplete#context()) | endif'
-            if type(l:event) == type('')
+            if type(l:event) == v:t_string
```

Comment:
older vim/neovim doesn't have support for this so I stopped using v:t_ and instead use `type` function only.